internals/cli/cmd_okay.go

```diff
@@ -23,20 +23,17 @@ import (
 	cmdpkg "github.com/canonical/pebble/cmd"
 )
 
-const cmdOkaySummary = "Acknowledge notices and warnings"
+const cmdOkaySummary = "Acknowledge notices"
 const cmdOkayDescription = `
-The okay command acknowledges warnings and notices that have been previously
-listed using '{{.ProgramName}} warnings' or '{{.ProgramName}} notices', so that they are omitted
-from future runs of either command. When a notice or warning is repeated, it
-will again show up until the next '{{.ProgramName}} okay'.
+The okay command acknowledges notices that have been previously listed using
+'{{.ProgramName}} notices', so that they are omitted from future runs of the command.
+When a notice is repeated, it will show up again until the next '{{.ProgramName}} okay'.
 `
 
 type cmdOkay struct {
 	client *client.Client
 
 	socketPath string
-
-	Warnings bool `long:"warnings"`
```

Comment:
This might break CLI-using scripts that used `--warnings` before. Should we leave this in as a no-op, but undocumented flag (so that scripts that pass `--warnings` continue to work) to not break existing CLI users? (assuming `go-flags` fails for unknown command-line flags, and doesn't just ignore them by default, anyway)